src/Esfa.Roatp.Register.ApiSystemTests/Features/RoatpRegisterApi.feature

```diff
@@ -13,10 +13,22 @@ Given the following roatp providers are available
 Scenario: Roatp Register API should return all providers
 When I request for All providers
 Then I should get All providers
+And returns All Providers UKPRN field
+And returns All Providers Name field
+And returns All Providers NewOrganisationWithoutFinancialTrackRecord field
+And returns All Providers ParentCompanyGuarantee field
+And returns All Providers ProviderType field
+And returns All Providers StartDate field
```

Comment:
I understand why you've done it like this - however, I'd personally prefer it to be contained within something like

```Then I get all provider information```

instead of

```
Then I should get All providers
And....
And...
And...
```

Then the assertion can check for each provider property is populated - happy at the end of the day for it to be done either way, important thing is that its been tested!

Reply:
changed step definition